src/psyclone/psyGen.py

```diff
@@ -3476,37 +3478,46 @@ def arguments(self):
     @property
     def name(self):
         '''
-        :returns: the name of the kernel associated with this call.
+        :returns: the name of the kernel.
```

Comment:
Capitalise

Reply:
well, looking at other docstrings they are not always capitalised, so I guess is more of a question, should they be?

Reply:
Strictly speaking, I don't think they should as the contents appear after a ":" in the generated documentation (https://psyclone-ref.readthedocs.io/en/latest/autogenerated/psyclone.dynamo0p3.html#psyclone.dynamo0p3.DynKernMetadata). I must admit I'm not too worried but we should probably be consistent at least.